RELEASE_NOTES

```diff
@@ -1,6 +1,23 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.2.0 (26 November 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* adds information on installing and using the smee client on `aarch64` (#233)
+
+Improvements:
+* support for running tests inside the same job but after the build step (#222)
+  * runs `bot/test.sh` and `bot/check-test.sh` if these are provided by the pull
+    request being processed
```

Comment:
I would rephrase this as:

```suggestion
  * runs `bot/test.sh` and `bot/check-test.sh` if these are provided by in the target repository
```

the fact that these scripts initially make part of the PR (or are updated in a PR) is a detail we don't need to mention here imho (since that applies to other stuff the bot picks up on in `bot/` too)

Reply:
Yes. It was an attempt to distinguish a bit from the "repository" later mentioned.

Reply:
Addressed in https://github.com/EESSI/eessi-bot-software-layer/pull/232/commits/7320c1a31d6af51ad8f6923adda91057d58e6702